tests/profile_fields_test.php

```diff
@@ -193,7 +191,16 @@ public function test_get_profile_fields() {
             'profile_field_text2' => 'Test text2',
         ];
 
-        $this->assertSame($expected, profile_fields::get_profile_fields());
+        $output = profile_fields::get_profile_fields();
```

Comment:
as we don't actually testing string itself, but more like that it gets assigned to a correct lang string, would it be better to modify expected array to something like:

```
        $expected = [
            'username' => get_string('username'),
            'firstname' => get_string('firstname'),
            ........
            'profile_field_text1' => 'Test text1',
            'profile_field_checkbox1' => 'Test checkbox1',
            'profile_field_checkbox2' => 'Test checkbox2',
            'profile_field_text2' => 'Test text2',
        ];
```

Reply:
thanks for reviewing @dmitriim, yep the get_string is better, I will change to it.